### PR TITLE
Exported to lambda, updated falafel, updated service, and minor version bump

### PR DESCRIPTION
**Pull request by Johnbastian**

### CODE REVIEW

File: connectors.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "bigquery",
     "title": "Google BigQuery",
     "description": "Schedule jobs and get query results via the Google BigQuery APi.",
-    "version": "1.0",
+    "version": "1.1",
```
Comment (chrisjhoughton) [inline connectors.json]: Doesn't need a version bump

File: package.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
   "description": "Schedule jobs and get query results via the Google BigQuery APi.",
   "version": "0.0.1",
   "dependencies": {
-    "@trayio/falafel": "^1.0.24"
+    "@trayio/falafel": "^1.2.5",
+    "express": "^4.14.0"
```
Comment (chrisjhoughton) [inline package.json]: This should be a dev dependency. On dev dependencies - copy them from another one into this connector

File: connectors.json (hunk 1 above)
Comment (Johnbastian) [inline connectors.json]: It's a service change... why won't it need a bump?

Comment (chrisjhoughton) [inline connectors.json]: my bad, you're right

**Comment by Johnbastian** [comment]
@alirussell @chrisjhoughton Is this good to go?

**Comment by chrisjhoughton** [comment]
LGTM

**Pull request merged**